ros/src/socket/packages/vehicle_socket/nodes/vehicle_sender/vehicle_sender.cpp

```diff
@@ -112,8 +112,7 @@ static void *sendCommand(void *arg)
   oss << command_data.brakeValue << ",";
   oss << command_data.steerValue << ",";
   oss << command_data.linear_velocity << ",";
-  oss << command_data.steering_angle << ",";
-  oss << command_data.lampValue;
```

Comment:
@s-azumi @yk-fujii Can this line be deleted? I'd like to know why and to split into another pull request.

Reply:
They told me that this should not be changed. I'll revert it.

Reply:
Reverted at 04b2a1b.